make/autoconf/toolchain.m4

```diff
@@ -217,25 +216,6 @@ AC_DEFUN_ONCE([TOOLCHAIN_DETERMINE_TOOLCHAIN_TYPE],
   # First toolchain type in the list is the default
   DEFAULT_TOOLCHAIN=${VALID_TOOLCHAINS%% *}
 
-  # On AIX the default toolchain depends on the installed (found) compiler
-  #   xlclang++     -> xlc toolchain
-  #   ibm-clang++_r -> clang toolchain
-  # The compiler is searched on the PATH and TOOLCHAIN_PATH
-  # xlclang++ has precedence over ibm-clang++_r if both are installed
-  if test "x$OPENJDK_TARGET_OS" = xaix; then
-    DEFAULT_TOOLCHAIN="clang"
-    if test "x$TOOLCHAIN_PATH" != x; then
-      if test -e ${TOOLCHAIN_PATH}/xlclang++; then
-        DEFAULT_TOOLCHAIN="xlc"
-      fi
-    else
-      UTIL_LOOKUP_PROGS(XLCLANG_TEST_PATH, xlclang++)
-      if test "x$XLCLANG_TEST_PATH" != x; then
-        DEFAULT_TOOLCHAIN="xlc"
-      fi
-    fi
-  fi
```

Comment:
We still need xlc support; this file should not change relative to 3ee63f4b08757014ffa7869d27f78b77852becf8.

Reply:
Reverted OpenJDK change.